### PR TITLE
feat: Add dedicated columns to name tables

### LINKED ISSUE #404: Add dedicated columns to name tables
Some columns in the names table have two lines but now that we have more space we could add dedicated columns (except for block and time columns).

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #404

## Demo

https://github.com/aeternity/aescan/assets/15363559/294b2b88-0ea4-4508-a25f-799878d477f1

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] Add dedicated columns to name tables

### CODE REVIEW

**Review by michele-franchi — COMMENTED** [review]
@janmichek Buyer, Highest Bid and Last Buyer columns should be separated but time and block should be one column as the other tables.

**Comment by janmichek** [comment]
> @janmichek Buyer, Highest Bid and Last Buyer columns should be separated but time and block should be one column as the other tables.

I tried to split the columns you mentioned, but also left the time-related columns as one line cell. SO everything is on one line. Coul you check, I think this would work

**Review by Liubov-crypto — COMMENTED** [review]
Should be Expired block or Expired Height? 

exp

the rest LGTM

**Review by michele-franchi — CHANGES_REQUESTED** [review]
As discussed, to be consistent with other tables, height and time columns should not be separate.

File: src/components/KeyblockDetailsPanel.vue [hunk 1]
```diff
@@ -256,11 +256,5 @@ const isNextKeyblockMined = computed(() =>
       }
     }
   }
-
-  &__keyblock-link--disabled {
```
Comment (janmichek) [inline src/components/KeyblockDetailsPanel.vue:260]: unrelated cleanup

**Comment by janmichek** [comment]
> Should be Expired block or Expired Height?
> 
> exp
> 
> the rest LGTM

Thanks, I reverted the change

**Comment by janmichek** [comment]
> As discussed, to be consistent with other tables, height and time columns should not be separate.

Reverted. I was curious how it's gonna feel like on line. At least we know that it could fit .)

File: src/utils/hints/namesHints.js [hunk 2]
```diff
@@ -2,15 +2,18 @@ export const namesHints = {
...
   nameInAuction: 'Unique identifier that may be configured to point to accounts, contracts, or oracles. names shorter than 13 characters are always auctioned.',
   expires: 'Keyblock height and estimated date and time when the name will be detached from the account. After expiration, the name enters into a revoked state and can be claimed again after a fixed timeout of 2016 keyblocks.',
-  buyer: 'Account that paid the shown price to claim the name.',
+  buyer: 'Account that paid claim the name.',
+  price: 'Price paid to claim the name.',
   owner: 'Owner of the name.',
   pointsTo: 'The account the name points to.',
-  highestBid: 'Account of the highest bidder and the highest bid price. Required minimum bid increase is 5%.',
+  highestBid: 'Account of the highest bidder.',
+  highestBidder: 'Highest bid price. Required minimum bid increase is 5%.',
```
Comment (michele-franchi) [inline src/utils/hints/namesHints.js:10]:
```suggestion
  highestBid: 'Highest bid price. Required minimum bid increase is 5%.',
  highestBidder: 'Account of the highest bidder.',
```

**Review by michele-franchi — APPROVED** [review]
Two hints are inverted but other than that it looks good.

File: src/utils/hints/namesHints.js (hunk 2 above)
Comment (janmichek) [reply src/utils/hints/namesHints.js]: Thanks for the suggestion. Accepted

**Pull request merged**